python/python/bystro/supervised_ppca/_base.py

```diff
@@ -66,6 +66,41 @@
 from bystro._template_sgd_np import BaseSGDModel
 
 
+def _get_projection_matrix(W_: Tensor, sigma_: Tensor):
+    """
+    This is currently just implemented for PPCA due to nicer formula. Will
```

Comment:
Is this comment still relevant, and if so, can we remove it and  make implementing it more broadly a Github issue?

Reply:
Someday we'll want to also include factor analysis which will have non-isotropic noise. Don't know how soon that will be necessary however. Up to you.